### DISCUSSION TITLE
How to Create a Custom Connector - `Keplr` / `Evmos`?

### DISCUSSION THREAD

**Discussion by peetzweg** [wevm/wagmi#430] [Q&A] [answered]
With `Evmos` main net launching, I want to support using the Keplr wallet in my dapp. How could I write a custom connector for it?

**Comment by peetzweg** [comment, marked as answer]
I guess this Injected/MetaMask is a similar connector to copy:
https://github.com/tmm/wagmi/blob/main/packages/core/src/connectors/injected.ts
https://github.com/tmm/wagmi/blob/main/packages/core/src/connectors/metaMask.ts

And these Keplr docs seem to a good start 🤔 

https://docs.keplr.app/api/

Recent example PR:
https://github.com/tmm/wagmi/pull/414/files

**Reply by tmm** [reply]
I created a guide for this https://wagmi.sh/examples/custom-connector
